bounties_api/notifications/email.py

```diff
@@ -136,6 +142,7 @@ def __init__(self, **kwargs):
                 # Rating for the fulfiller from the issuer
                 ratings = user_reviewees.filter(
                     fulfillment_review__isnull=False)
+                rating_link += '&fulfiller=true'
```

Comment:
Note that https://github.com/Bounties-Network/Explorer/pull/217 now makes it possible to say `&issuer=true` here, but because that's the default, I'm keeping it simple here.